File: VB/WinWebSolution.Module.Web/WebSortListViewController.vb
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base
Imports DevExpress.ExpressApp.Web.Editors.ASPx

Namespace WinWebSolution.Module.Win
	Public Class WinSortRootListViewController
		Inherits SortListViewControllerBase

		Protected Overrides Sub OnViewControlsCreated()
			MyBase.OnViewControlsCreated()
			Dim gridListEditor As ASPxGridListEditor = TryCast(View.Editor, ASPxGridListEditor)
			If gridListEditor IsNot Nothing Then
				gridListEditor.Grid.SettingsBehavior.AllowSort = False
			End If
		End Sub
	End Class
End Namespace

File: CSOld/WinWebSolution.Module.Web/WebSortListViewController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;
using DevExpress.ExpressApp.Web.Editors.ASPx;

namespace WinWebSolution.Module.Win {
    public class WinSortRootListViewController : SortListViewControllerBase {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            ASPxGridListEditor gridListEditor = View.Editor as ASPxGridListEditor;
            if (gridListEditor != null) {
               // gridListEditor.Grid.SettingsBehavior.AllowSort = false;
            }
        }
    }
}